src/pages/de/blog/erfahrungsbericht-der-bestell-app-in-einem-restaurant/index.mdx

```diff
@@ -0,0 +1,95 @@
+
+## Meine Eindrücke
+
+Um einen gewissen Kontext herzustellen: wir waren eine Gruppe von vier Personen, deshalb spreche ich immer von "wir".
+
+Es war ein wunderschöner sonniger Tag und am Abend waren wir bereit für dieses einzigartige Erlebnis. Sobald wir das Restaurant betraten, bemerkte ich die freundliche Atmosphäre des Restaurants und die Musik im Hintergrund.
+
+Gleich wurden wir vom Personal begrusst, das sofort unsere Reservierung aufnahm und uns zu unserem Tisch führte. Eine ganz normale Erfahrung aus dem Alltag in einem Restaurant.
+
+Dann setzten wir uns an den Tisch und wurden noch gefragt, ob wir ein Tablet benutzen wollten. Die Einführung eines Tablets zum Lesen des QR Codes bleibt dem Restaurant überlassen.
+
+Es scheint kompliziert zu sein, aber mit der Kamera des Handys können wir den QR-Code auf dem Tisch scannen und haben so Zugriff auf die digitale Speisekarte auf dem Handy. Wir haben uns für unsere Handys entschieden, weil es für uns bequemer war.
+
+Mit unserem Handys hatten wir Zugang zu allen Getränken und Speisen, die das Restaurant anbot. Es war auch möglich, die Menge des Getränks zu bestimmen. 5dl oder 3dl?
+
+Wir begannen dieses Erlebnis mit der Bestellung einer Vorpeise und von Getränken. Mit einem Klick und schon hatten wir unsere Bestellung aufgegeben. Ein paar Minuten später kam das Personal mit unserer Bestellung und stellte sie auf den Tisch. Woah, das ging unglaublich schnell. Nach der Vorspeise bestellten wir über die Bestell-App auch einen Hauptgang. So haben wir es auch mit unseren Kaffees am Ende gemacht.
+
+Für mich einer der bemerkenswertesten Vorteile einer Bestell-App ist die Schnelligkeit, mit der wir unsere Bestellungen bearbeiten konnten. Das Ergebnis war eine bequeme Art, das Restaurant zu besuchen, ohne dass wir auf das Personal warten mussten.
+
+Ein weiterer Punkt, der mir aufgefallen ist, war die Interaktion zwischen den Gästen und dem Personal. Meiner Meinung nach gab es keinen Abbruch des direkten Kontakts zwischen den Gästen und dem Personal. Die Verwendung von Handys im Alltag ist bereits eine Realität. Das Hinzufügen von Bestellungen und Zahlungen über das Handy war entsprechend natürlich.
+
+Darüber hinaus ist in dieser Bestell-App ein spezieller Button integriert. Mit dem können die Gäste das Personal rufen, um Fragen zu klären. Diese Funktion ist spezielle für das von uns besuchte Restaurant.
+
+## Ein tolles Erlebnis
+
+Nachfolgend ein Vergleich von zwei Situationen, die ich erlebt habe, und deren Ergebnis für mich als Gast:
+
+### Ohne Bestell-App
+
+Während meines Urlaubs bei einem meiner letzten gastronomischen Erlebnisse im Ausland beschloss ich, in ein Restaurant zu gehen, das gute Reviews hatte. Ich wartete etwa 30 Minuten auf die Speisekarte und weitere 30 Minuten auf das Essen. Eine Bestell-App hätte mir einen viel positiveren Eindruck von dem Restaurant vermittelt und die Wartezeit wäre nicht nötig gewesen.
+
```

Comment:
Die würde ich in einem separaten Bericht erzählen und nicht im Fazit erwähnen. Im Fazit nur auf das Bezug nehmen, was bereits geschrieben wurde.